src/main/scala/zio/prelude/package.scala

```diff
@@ -61,6 +62,9 @@ package object prelude
   type DeriveInverse[F[_]]     = Derive[F, Inverse]
   type DeriveOrd[F[_]]         = Derive[F, Ord]
 
+  type Divariant[:=>[-_, +_]]   = Zivariant[({ type lambda[-R, +E, +A] = R :=> A })#lambda]
```

Comment:
This raises the question, whether we should encode mere `Covariant` and `Contravariant` as special cases of `Zivariant`.
:thinking: 
Perhaps that's something we don't need to decide right now and can leave this to a future PR.

Reply:
Yes :)

There are few things to consider when pushing this encoding further:
- laws for Bicontravariant, laws for Divariant
- Covariant/Contravariant extends some other types and have extra methods, add them in syntax or expand the hierarchy of Trivariant 
- use asInstanceOf instead of deriving (currently it does not work b/c Covariant/Contravarian are not type aliases)
- encode the same thing with Invariant

Those will be pretty big PR's, hard to review if done at once.